sonarlint/sonarlint.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>sonarlint</id>
+    <title>SonarLint VSIX Extension</title>
+    <version>2.2.1</version>
+    <authors>SonarSource (SonarSource SA)</authors>
+    <owners>IOZ AG</owners>
+    <projectUrl>http://vs.sonarlint.org/</projectUrl>
+    <packageSourceUrl>https://github.com/iozag/chocolatey-packages/tree/master/git-lfs-install</packageSourceUrl>
+    <licenseUrl>http://opensource.org/licenses/MIT</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>SonarLint provides on-the-fly feedback to developers on new bugs and quality issues injected into C# and VB.Net code. In addition the connected mode allows to enforce governance policies by reporting the same issues in Visual Studio and in SonarQube server.</summary>
+    <description>SonarLint is a Visual Studio 2015 extension that provides on-the-fly feedback to developers on new bugs and quality issues injected into C# and VB.NET code. SonarLint for Visual Studio is based on and benefits from the .NET Compiler Platform ("Roslyn") and its code analysis API to provide a fully-integrated user experience in Visual Studio 2015. With version 2.0 you can now establish a connection to a SonarQube server and bind your Visual Studio solution to a SonarQube project. This operation automatically updates the rulesets of the solution and attaches the required Roslyn analyzers to the solution.</description>
+    <tags>SonarLint SonarQube SonarSource VisualStudio</tags>
+    <releaseNotes>https://github.com/SonarSource-VisualStudio/sonaranalyzer-csharp/releases/releaseNotes>
```

Comment:
This are the release notes for the analyzer, not for SonarLint. Mabye http://www.sonarlint.org/visualstudio/#News is the best link.